src/SlashCommands.tsx

```diff
@@ -139,9 +139,17 @@ export class Command {
         return this.getCommand() + " " + this.args;
     }
 
-    run(roomId: string, args: string) {
+    run(roomId: string, threadId: string, args: string) {
```

Comment:
explicit member access would be great 😇

Reply:
I'm not sure what that would look like in this context? Could you expand a bit?